internal/http/services/owncloud/ocs/conversions/permissions.go

```diff
@@ -39,21 +39,30 @@ const (
 	PermissionDelete
 	// PermissionShare grants share permissions on a resource
 	PermissionShare
+	// PermissionDeny grants permissions to deny access on a resource
+	// The recipient of the resource will then have PermissionNone.
+	PermissionDeny
+	// PermissionNone grants no permissions on a resource
+	PermissionNone
+	// PermissionMax is to be used within value range checks
+	PermissionMax Permissions = (1 << (iota - 1)) - 1
```

Comment:
`PermissionMax` makes no sense. It would include `PermissionNone` which would take away all other permissions

Reply:
See my comment below, it is used for validation purposes. Maybe you have a problem with the naming.

Should it be `PermissionMaxInput` ?

Reply:
Yes, that is much clearer :+1: